nlp/drivers/nlp_auglag.py

```diff
@@ -50,7 +50,7 @@ def auglag_stats(auglag):
 for problem in sys.argv[1:]:
     model = PySparseAmplModel(problem)
     model.compute_scaling_obj()
-    auglag = Auglag(model, TRON, maxiter=100)
+    auglag = Auglag(model, TRON, maxtime=900)
```

Comment:
The time idea was more for benchmarking. I changed it back to the number of times the penalty parameter or Lagrange multiplier is updated - see the latest commits.